<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proxy Verify</string>
  </property>
  <property name="windowIcon">
   <iconset resource="proxy.qrc">
    <normaloff>:/new/icon/proxy.png</normaloff>:/new/icon/proxy.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <widget class="QLineEdit" name="ip">
           <property name="geometry">
            <rect>
             <x>23</x>
             <y>0</y>
             <width>131</width>
             <height>27</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="port">
           <property name="geometry">
            <rect>
             <x>192</x>
             <y>0</y>
             <width>71</width>
             <height>27</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="portPanel">
           <property name="geometry">
            <rect>
             <x>158</x>
             <y>4</y>
             <width>31</width>
             <height>17</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>0~65535</string>
           </property>
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
          <widget class="QLabel" name="ipPanel">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>5</y>
             <width>14</width>
             <height>17</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>0.0.0.0~255.255.255.255</string>
           </property>
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
          <widget class="QPushButton" name="button_add">
           <property name="geometry">
            <rect>
             <x>267</x>
             <y>0</y>
             <width>51</width>
             <height>27</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
          <widget class="QPushButton" name="button_del">
           <property name="geometry">
            <rect>
             <x>327</x>
             <y>0</y>
             <width>61</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
          <widget class="QPushButton" name="button_clear">
           <property name="geometry">
            <rect>
             <x>397</x>
             <y>0</y>
             <width>51</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="Custom_TableWidget_Site" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>160</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Port</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Message</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QPushButton" name="button_import">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Import</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_paste">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Paste from clipboard!
e.g.:127.0.0.1:8080</string>
          </property>
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_download">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>dl from URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="https">
          <property name="toolTip">
           <string>Check https protocol support.</string>
          </property>
          <property name="text">
           <string>Https</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_verify">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Verify all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_quit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Custom_TableWidget_Site</class>
   <extends>QTableWidget</extends>
   <header>custom_tablewidget_site.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="proxy.qrc"/>
 </resources>
 <connections/>
</ui>
